_includes/head.html

```diff
@@ -17,5 +17,5 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
-
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<link rel="stylesheet" href="{{ base_path }}/assets/css/primer.css"
```

Comment:
> There are some docs here too: https://github.com/styleguide/css/getting-started#installing-via-npm

As much as I ❤️  NPM, we probably wouldn't want to add via NPM as it would be the only package, everything else is handled via Jekyll or [bower](https://github.com/bower/bower).